pkgs/servers/minio/default.nix

```diff
@@ -3,12 +3,12 @@
 stdenv.mkDerivation rec {
   name = "minio-${shortVersion}";
 
-  shortVersion = "20170613";
-  longVersion = "2017-06-13T19-01-01Z";
+  shortVersion = "20170805";
+  longVersion = "2017-08-05T00-00-53Z";
 
   src = fetchurl {
     url = "https://github.com/minio/minio/archive/RELEASE.${lib.replaceStrings [":"] ["-"] longVersion}.tar.gz";
```

Comment:
`replaceStrings` is not needed, just use `${version}`.

Reply:
Done